README.md

```diff
@@ -1,6 +1,164 @@
 # NCIT Alumni Portal
+
 Welcome to NCIT Alumni Portal.
 
+<!-- cover photo here -->
+
+## More than a Website - An Astro Theme
+
+This project is not just an Alumni Portal it is also an open source template for creating your own Alumni Portal or a Landing Page for your community. This project is built using [Astro](https://astro.build/), and available in the Astro theme section.
+
+**[View Live Demo](https://ncitalums.com/)**
+
+## Table of Contents
+
+<!-- * [What's New](#whats-new) -->
+
+- [Features](#features)
+- [Getting Started](#getting-started)
+- [Deployment](#deployment)
+- [Project Structure](#project-structure)
+- [Customization](#customization)
+- [Contributing](#contributing)
+
+## Features
+
+- **Astro Features**: A fast static site builder that is developer-friendly, **SEO**-friendly, and supports modern web technologies like React, Vue, Svelte, Tailwind CSS, etc.
+- **Uniform & Responsive**: A responsive design with uniform style, use of colors, typography, and components layout across the site.
+- **Multiple Applications**: A site that can be used for multiple purposes like Landing Page, Alumni Portal, Community Portal, Blog, etc.
+- **Plug & Play**: A template where each component, each blog and the whole page can that can be easily customized and extended.
+
+## Getting Started
+
+This guide will provide you with the necessary steps to set up and familiarize yourself with the Alumni Portal on your local development machine.
+
+### Use This Template
+
+To get started, click the `Use this template` button (the big green one at the top right) to create your own repo from this template in your GitHub account. _If the button is not visible_, you can clone this repository to your local machine and create a new repository in your GitHub account. Don't forget to delete the `.git` folder to start fresh when pushing to your own repository.
+
+### Clone the Repository
+
+Once your repository is created, you can clone it to your local machine using the following commands:
+
+```bash
+git clone https://github.com/[YOUR_USERNAME]/ncitalums.git
+cd [YOUR_REPO_NAME]
+```
+
+### Installation
+
+Start by installing the project dependencies. Open your terminal, navigate to the project's root directory, and execute:
+
+```bash
+npm install
```

Comment:
@axyut I saw you get mixed up with yarn and NPM while we have used yarn for this project. Please consider updating these to yarn instead of NPM.

Reply:
sure